docs/source/telegram.constants.rst

```diff
@@ -4,3 +4,4 @@ telegram.constants Module
 .. automodule:: telegram.constants
     :members:
     :show-inheritance:
+    :member-order: bysource
```

Comment:
I would suggest either renaming the constants then - or keeping `bysource` in `telegram.constants.rst` but ditch it in classes because there the amount of constants is small

Reply:
OK, I'll ditch `bysource` in classes. @Bibo-Joshi what do you think about keeping it in `constants.py`? At least when I'm adding classes there, I try to maintain the alphabetical order anyway

Reply:
I don't see the benefit of having the `bysource` thingy for the order of the classes. Having the classes sorted alphabetically in the code certainly is good, but if we don't have to remember that, I'd prefer that …

Reply:
OK. Removing `bysource` from `constants.py`.